kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -1285,79 +1285,106 @@ def deprovision(self, req):
+                logger.info('openvswitch already stopped, skipping')
 
-        # 6. Stop ovsdb-server if it is a separate service unit
-        r, o, e = bash.bash_roe('systemctl stop ovsdb-server')
-        if r != 0:
-            logger.info('ovsdb-server.service not available, '
-                        'already stopped by openvswitch')
-        else:
-            logger.info('stopped ovsdb-server')
+            # 6. Stop ovsdb-server if it is a separate service unit
+            if _is_service_active('ovsdb-server'):
+                r, o, e = bash.bash_roe('systemctl stop ovsdb-server')
```

Ruff (0.15.6)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)